front/app/containers/InitiativesIndexPage/InitiativeInfoContent/index.tsx

```diff
@@ -41,7 +41,7 @@ const InitiativeInfoContent = memo<InjectedLocalized & Props>(
     const { data: appConfig } = useAppConfiguration();
 
     if (!isNilOrError(appConfig)) {
-      const voteThreshold =
+      const reactionThreshold =
         appConfig.data.attributes.settings.initiatives?.voting_threshold;
```

Comment:
This attribute is still called `voting_treshold`

Reply:
Yes, I've been on the fence as to whether to change this (because votes on proposals will still be called votes) but in retrospect as this is part of the codebase not the frontend I think I should, given that I changed everything that references it.